oaknational/Oak-Web-Application
# [CUR-261] Updated 'intent' to 'explainer' on overview page

## Description

- Updated the title of curriculum intent to curriculum explainer
- Updated font sizes for all headings across overview tab and curriculum header


## How to test

1. Go to https://deploy-preview-1914--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Go to any subject/phase combination
3. You should see 'Curriculum explainer' instead of 'Curriculum intent'

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-09-19 at 17 14 03](https://github.com/oaknational/Oak-Web-Application/assets/32605982/7f36993f-e517-4ff0-97d9-1ca76d0ceabe)

How it should now look:
![Screenshot 2023-09-19 at 17 13 37](https://github.com/oaknational/Oak-Web-Application/assets/32605982/5e24b547-12c0-4404-a040-0fc353f03725)

## Checklist
